source/adapters/level_zero/sampler.cpp

```diff
@@ -75,9 +76,9 @@ UR_APIEXPORT ur_result_t UR_APICALL urSamplerCreate(
       ZeSamplerDesc.addressMode = ZE_SAMPLER_ADDRESS_MODE_MIRROR;
       break;
     default:
-      urPrint("urSamplerCreate: unsupported "
-              "UR_SAMPLER_PROPERTIES_ADDRESSING_MODEE "
-              "value\n");
+      logger ::error("urSamplerCreate: unsupported "
```

Comment:
nit: Remove a whitespace after the namespace name.

Reply:
Dpne